File: 1-easy/problems/2011-final-value-of-variable-after-performing-operations/solution.ts
function finalValueAfterOperations(operations: any): boolean | number | string | any {
  """
  :type operations: List[str]
  :rtype: int
  """
  return sum(1 if '+' == op[1] else -1 for op in operations);
}

export { finalValueAfterOperations };

File: 1-easy/problems/2011-final-value-of-variable-after-performing-operations/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def finalValueAfterOperations(self, operations):
        """
        :type operations: List[str]
        :rtype: int
        """
        return sum(1 if '+' == op[1] else -1 for op in operations)
